src/app/message/message.component.ts

```diff
@@ -132,6 +132,11 @@ export class MessageComponent implements OnInit, OnDestroy {
       .subscribe(() => {
         this.sending = false;
         this.currentMessage = '';
+        // Waiting for DOM to update model 'this.currentMessage'
+        // to correctly resize the textarea
+        setTimeout(() => {
```

Comment:
Things got messy here. Didn't find another way which would resize the textarea. Without the timeout it doesn't work. This may be a bug which will be fixed in a future version of material2.